### PR TITLE
Added the documentation for django

### LINKED ISSUE #148: Write documentation for running the Django application
To follow up on today's class discussion, we should try to make sure that everyone in the class is able to run our application and its test suite. In particular, it would be useful to have some concise documentation in our project which explains how to start the Django dev server and perform any common setup tasks. Ideally, this documentation could also include some information about how to configure Django, as well as about the structure of the project (e.g. the model / view / controller design pattern).

### PR DESCRIPTION
**Pull request by rauniyars**
# Description of Pull Request

This PR adds the documentation to show how to run and test the Django application, it also includes 
some information about the structure of the project.

This PR fixes #148 

## Type of Change

Please describe the pull request as one of the following: 

- [ ] Bug fix
- [ ] Breaking change
- [ ] New feature
- [x] Documentation update
- [ ] Other (Please specify any helpful information below)

## Tags

@rauniyars @barrezuetai @enpuyou

### CODE REVIEW

**Comment by Michionlion** [comment]
This PR will not be approved if the PR Template is not filled out. @rauniyars you must edit your PR description and actually add text.

File: djando_documentation.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#### Django
+
+Django is a free and open source web application framework that is written in Python and helps to implement websites in a convenient manner. With the collection of modules embedded within its framework, development is made easier and one would not have to work on creating the website from scratch but just use the components that are already built within which saves a lot of time. Acknowledging the fact that we have a short span of time to work on the GatorGrouper project, we decided to use Django as it provides the developer with simpler tools to grasp the project's structure. In addition to this, it supports Object-Relational Mapping, which allows the user to create a bridge between the data model and the database engine, and supports a large set of database systems including SQLite.
+
+#### Running
+
+After the successful installation of Django, we inclined towards running the application. In order to run the application smoothly, please make sure that you are in the correct directory which is GatorGrouper_Django.
```
Comment (Michionlion) [inline djando_documentation.md]: "inclined towards" doesn't make any sense.

Additionally, `GatorGrouper_Django` is no longer the correct folder.

File: djando_documentation.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Next, make sure to run this command in the terminal: `python manage.py runserver`
```
Comment (Michionlion) [inline djando_documentation.md]: This does not mention the required `pipenv` usage.

File: djando_documentation.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+To check if your server is set up and running, open up your browser and type: `local host:80000`
```
Comment (Michionlion) [inline djando_documentation.md]: This is not the correct URL.

File: djando_documentation.md (hunk 2 above)
Comment (enpuyou) [reply djando_documentation.md]: Fixed in the most recent commits. Thanks!

File: djando_documentation.md (hunk 3 above)
Comment (enpuyou) [reply djando_documentation.md]: Fixed in the most recent commits. Thanks!

**Comment by Lancasterwu** [comment]
Hi @Michionlio, I think this should be merged into the master.

**Pull request merged**